addons/account/tools/certificate.py

```diff
@@ -50,8 +50,8 @@ def public_numbers():
                 for item in certificate.subject.get_attributes_for_oid(oid)
             ]
         ),
-        not_valid_after=certificate.not_valid_after,
-        not_valid_before=certificate.not_valid_before,
+        not_valid_after=certificate.not_valid_after_utc,
+        not_valid_before=certificate.not_valid_before_utc,
```

Comment:
From the docs, this doesn't exist in `cryptography < 42.0.0`, but we are using a lower version for python < 3.12 :thinking:

Reply:
Thanks nice catch.